apps/qaqc/src/checkbook/checkbook.py

```diff
@@ -31,17 +17,10 @@ def checkbook():
         """
     )
 
-    st.dataframe(data)
-
-    grouped_by_category_all = get_category_data(data)
-    grouped_by_geometry_all = get_geometry_data(data)
-
-    st.header("Check Amounts Per Category")
-    st.bar_chart(grouped_by_category_all, x="final_category", y="check_amount")
-
-    agency = st.selectbox(
+    agency = st.sidebar.selectbox(
         "Agency Selection: ",
         (
+            "All",
```

Comment:
just a future thought: would be cool to generate the list of options from the values in the data

Reply:
Agreed. First though there's some data cleanup in the checkbook process that's needed right now, as these are lists in the dataset (aggregated across checks)